### ISSUE TITLE
Retry Button for Failed Releases not visible on mobile

### ISSUE THREAD

**Issue by bakerboy448** [autobrr/autobrr#1560]
**Version**
The autobrr version you're running. If you run something other than latest, please update before reporting issues.

Version
v1.42.0
Commit
7134e06379ae2a77e597904e6a2014536d358b7a
Build date
2024-05-09T09:17:26.371Z

**Describe the bug**
Can't access retry botton as the error pushes it away

**To Reproduce**
1. Go releases
2. Click on a failed push
3. Can't retry

**Expected behavior**
Can click retry on mobile

**Screenshots**
![image](https://github.com/autobrr/autobrr/assets/55419169/e5478c85-b037-410a-b66b-7df737dee788)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6] iPhone 14P
 - OS: [e.g. iOS8.1] 17.4.1
 - Browser [e.g. stock browser, safari] chrome
 - Version [e.g. 22] 125.0.6422.51

**Additional context**

**Comment by martylukyy** [comment]
Just checked this on Google Chrome on Android and the iOS 18 beta with 1.45.0 
and i can see the retry button on both rejected and errored actions.
Closing this since it seems like this got fixed in the meanwhile and this issue has not been closed.

Feel free to reopen this issue in case it has not been fixed in your case @bakerboy448 !
